### PR TITLE
Added BigDataCloud API

### PR DESCRIPTION
**Pull request by bexdeep**
BigDataCloud provides various paid and free-to-use geolocation APIs for websites and mobile apps.

### CODE REVIEW

File: src/technologies/b.json [hunk 1]
```diff
@@ -635,6 +635,22 @@
     "url": "mybigcommerce\\.com",
     "website": "http://www.bigcommerce.com"
   },
+  "BigDataCloud IP Geolocation": {
+    "cats": [
+      79
+    ],
+    "description": "BigDataCloud IP Geolocation API provides detailed and accurate locality and security metrics of an IP address.",
+    "icon": "BigDataCloud-IPGeolocation.svg",
```
Comment (nurbek91) [inline src/technologies/b.json:643]: BigDataCloud-IPGeolocation
32x32

**Comment by nurbek91** [comment]
examples:
xhr
https://vocab1.com/
https://cititex.com/
missed
https://www.giglon.com/

File: src/technologies/b.json (hunk 1 above)
Comment (AliasIO) [reply src/technologies/b.json:643]: @bexdeep Mind updating your PR with this icon?

Comment (bexdeep) [reply src/technologies/b.json:643]: Hi @AliasIO , Thanks for doing this. Can I upload something else of the same size though? I think they use only cloud logo.

Comment (AliasIO) [reply src/technologies/b.json:643]: Yes, it just needs to be a square SVG. Thanks!

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**